datalad/interface/create_collection.py

```diff
@@ -12,17 +12,44 @@
 
 __docformat__ = 'restructuredtext'
 
+
+from os import curdir
+from os.path import join as opj, abspath, expandvars, expanduser
 from .base import Interface
 from datalad.support.param import Parameter
-from datalad.support.constraints import EnsureStr
+from datalad.support.constraints import EnsureStr, EnsureNone
+from datalad.support.collectionrepo import CollectionRepo
+from appdirs import AppDirs
+
+dirs = AppDirs("datalad", "datalad.org")
```

Comment:
I see this pattern over and over again -- I guess we need some kind of 'datalad.dirs' with some helpers like `get_user_data_path(filename)` which would do the magical import, AppDirs, opj?